demos/common/cpu_extensions_base/README.md

```diff
@@ -0,0 +1,7 @@
+These are generated files by the following command:
```

Comment:
Extensions generator has been removed from OpenVINO :(

Reply:
Yes, thanks. It's not a problem to replace - code is just too old :)

There is some FFT implementation here: https://github.com/dkurt/openvino_pytorch_layers/ but I have not tried to apply to this PR

Reply:
Updated CPU extensions code